api/src/controllers/server.ts

```diff
@@ -47,6 +47,20 @@ router.get(
 	})
 );
 
+router.get(
+	'/specs/ts',
+	asyncHandler(async (req, res) => {
+		const service = new SpecificationService({
+			accountability: req.accountability,
+			schema: req.schema,
+		});
+
+		res.type('text/plain');
+		res.send(await service.typescript.generate());
```

Comment:
Should this route prompt a file download like `/specs/graphql`?

If so, this should be added:
```suggestion
		res.attachment('types.ts');
		res.type('text/plain');
		res.send(await service.typescript.generate());
```

Reply:
Yes, lets do that 👍🏻